tests/test_crs.py

```diff
@@ -18,15 +16,15 @@
 # When possible, Rasterio gives you the CRS in the form of an EPSG code.
 def test_read_epsg(tmpdir):
     with rasterio.open('tests/data/RGB.byte.tif') as src:
-        assert src.crs == {'init': 'epsg:32618'}
+        assert src.crs.data == {'init': 'epsg:32618'}
```

Comment:
Yeah, I hadn't really intended for `crs.data` to be used as a public interface, but I guess that if I'm using it in the tests, then it is. `to_dict` sounds like a reasonable idea, though I would think it best to be a method rather than a property (to_ implies that some action is being taken).